### PR TITLE
Update error messages.

### PR DESCRIPTION
**Pull request by powerlanguage**
screenshot 2015-01-12 13 20 31

I wasn't sure where best to upload the new image.

The text was just whatever came to mind.  Feedback welcome.

:eyeglasses: @spladug @rram

### CODE REVIEW

File: 502.html [hunk 1]
```diff
@@ -2,10 +2,10 @@
...
+body{text-align:center;position:absolute;top:50%;margin:0;margin-top:-275px;width:100%}
+h2,h3{color:#555;font:bold 200%/100px sans-serif;margin:0}
 h3{color:#777;font:normal 150% sans-serif}
 </style>
-<img src=//www.redditstatic.com/youbrokeit1.png alt="">
-<h2>we took too long to make this page for you</h2>
-<h3>try again and hopefully we will be fast enough this time.
+<img src=//www.redditstatic.com/trouble-afoot.png alt="">
+<h2>sorry, something broke on our end</h2>
+<h3>please try again in a minute.</h3>
```
Comment (spladug) [inline 502.html]: minor request: the `.` in the second sentence on these error pages looks very very strange to me for some reason and i regret adding the original one. could it die please? :)

**Comment by spladug** [comment]
Perhaps a `<p>(error code: 503)</p>` down at the bottom?

File: 503.html [hunk 2]
```diff
@@ -6,6 +6,6 @@
 h2,h3{color:#555;font:bold 200%/100px sans-serif;margin:0}
 h3{color:#777;font:normal 150% sans-serif}
 </style>
-<img src=//www.redditstatic.com/heavy-load.png alt="">
-<h2>we took too long to make this page for you</h2>
-<h3>try again and hopefully we will be fast enough this time.
+<img src=//www.redditstatic.com/trouble-afoot.png alt="">
+<h2>no server was available to handle the request</h2>
```
Comment (spladug) [inline 503.html]: how about `all of our servers are busy right now`?

File: 502.html (hunk 1 above)
Comment (powerlanguage) [inline 502.html]: definitely agree.  wasn't sure who the original perpetrator was :)

**Comment by Deimos** [comment]
If we're updating these pages, should we add a link to redditstatus.com on them? Not sure if it's relevant on all of them, but it might be good to have.

**Comment by spladug** [comment]
I'm not sure, @Deimos. It might just be confusing for transient errors and cause us more headache of weird reports.

**Comment by powerlanguage** [comment]
screenshot 2015-01-13 11 32 18

:haircut:

**Comment by spladug** [comment]
:fish:!

**Comment by powerlanguage** [comment]
Squashed up.  Could benefit from a sanity test before deploying  I wasn't sure how to test they pages on my install.

**Comment by rram** [comment]
https://github.com/reddit/error-pages/commit/7e94c973e1d03da99f6d5f1391ee3166c67f830b

**Pull request closed without merging**